crates/bevy_reflect/src/impls/std.rs

```diff
@@ -708,13 +708,17 @@ impl<T: FromReflect> Reflect for Option<T> {
                         let field = value
                             .field_at(0)
                             .expect("Field at index 0 should exist")
```

Comment:
should this error message be updated like the one below?

Reply:
Yeah that's probably a good idea. I'll update it

Reply:
Updated the error messages!